src/go/rdctl/cmd/snapshotList.go

```diff
@@ -81,10 +82,25 @@ func tabularOutput(snapshots []snapshot.Snapshot) error {
 		return nil
 	}
 	writer := tabwriter.NewWriter(os.Stdout, 0, 4, 4, ' ', 0)
-	fmt.Fprintf(writer, "NAME\tCREATED\n")
+	fmt.Fprintf(writer, "NAME\tCREATED\tDESCRIPTION\n")
 	for _, aSnapshot := range snapshots {
 		prettyCreated := aSnapshot.Created.Format(time.RFC1123)
-		fmt.Fprintf(writer, "%s\t%s\n", aSnapshot.Name, prettyCreated)
+		desc := aSnapshot.Description
+		idx := strings.Index(desc, "\n")
+		if idx >= 0 {
+			// If the description starts with a newline, it will appear empty in this view.
+			// Use the json view to get the full description
+			desc = desc[0:idx]
+		}
+		if len(desc) > 63 {
+			desc = desc[0:60] + "..."
```

Comment:
And should we use a proper unicode ellipsis instead of appending 3 dots?

```suggestion
		if len(desc) > 61 {
			desc = desc[0:60] + "…"
```

The `60` feels somewhat arbitrary. At least when the output is a terminal, I would expect the output to use the full terminal width and not have a random cutoff point.

And I could make an argument that we should not truncate the output when it goes to a file (e.g. when you pipe into `less`) because the user can view the wrapped output, or scroll horizontally.

But we should open an enhancement request for this for 1.12.

Reply:
> But we should open an enhancement request for this for 1.12.

I've created #5782 for this.